game/script.rpy

```diff
@@ -55,8 +55,33 @@ label start:
 
     e "This information screen can also be opened from the menu."
 
+    e "You now have to make a decision within a limited time."
+
+    $ list_of_choices = [ ("choice 1", "result1"), ("choice 2", "result2"), ("choice 3", "result3") ]
+
+    show screen death_or_live(5, "fake_death")
+
+    $ result = renpy.display_menu(list_of_choices)
+
+    $ renpy.block_rollback()
```

Comment:
I thought it would be blocked with that, but the block command in the screen code only blocks rollback if you run out of time. I've tried moving the block command higher within the screen, in case it doesn't activate unless the timer runs out. But in both cases, the game lets you rollback and change your choice as long as you select an option within the time limit.
So we can either keep things as they are, or remove the block command in the code screen, and remember to use this both after the time-limited choices and at the beginning of the fail_part.

Reply:
I just tried it out, and surprisingly, it's already enabled. So we don't need to enable it after each `death_or_live` screen.